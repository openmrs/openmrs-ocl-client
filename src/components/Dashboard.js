import React from 'react';
import SideNavigation from './SideNavigation';
/**
 * Component for rendering the main page
 */
const Dashboard = () => (
  <div>
    <SideNavigation />
  </div>
);

export default Dashboard;
