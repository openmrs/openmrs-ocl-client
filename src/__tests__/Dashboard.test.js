import React from 'react';
import { mount } from 'enzyme';
import Dashboard from '../components/Dashboard';

const wrapper = mount(<Dashboard />);

describe('Dashboard Component', () => {
  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
