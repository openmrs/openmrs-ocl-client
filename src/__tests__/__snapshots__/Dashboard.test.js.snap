// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Component should render without crashing 1`] = `
<Dashboard>
  <div>
    <SideNavigation>
      <div
        className="sidenav"
      />
    </SideNavigation>
  </div>
</Dashboard>
`;
